widgets/views/wallEntry.php

```diff
@@ -62,21 +62,25 @@
     <?php if ($calendarEntry->canRespond()): ?>
        <div class="row" style="padding-top:10px">
             <div class="col-md-12">
-                <?= Button::defaultType(Yii::t('CalendarModule.views_entry_view', "Attend"))->sm()
-                    ->icon($participantSate === CalendarEntryParticipant::PARTICIPATION_STATE_ACCEPTED ? 'fa-check' : null)
-                    ->action('calendar.respond', $contentContainer->createUrl('/calendar/entry/respond', ['type' => CalendarEntryParticipant::PARTICIPATION_STATE_ACCEPTED, 'id' => $calendarEntry->id]))?>
 
+                <?php
+                $pButton = function($state, $label) use ($calendarEntry, $participantSate, $contentContainer) {
+                    return Button::defaultType(Yii::t('CalendarModule.views_entry_view', $label))->sm()
```

Comment:
This does not work, since the translation parser does not recognize this, the `Yii::t` statements must contain the actual text string not a variable.